### PR TITLE
docs: patch fix the edge tutorial

### PR DESCRIPTION
**Pull request by sudkul87**
## Describe the Change

This PR applies patch fixes to the newly published edge tutorial based on the Slack feedback Sudhanshu received. 

## Review Changes

💻 [Preview URL](https://deploy-preview-1409--docs-spectrocloud.netlify.app/clusters/edge/site-deployment/deploy-cluster)

### CODE REVIEW

File: content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md [hunk 1]
```diff
@@ -120,17 +120,46 @@ git checkout v3.4.3
 ```
 <br />
 
-## Review Arguments
+## Define Arguments
 
-The **.arg.template** sample file in the current directory contains customizable arguments, such as image tag, registry, repository, and OS distribution. Rename the sample **.arg.template** file to **.arg** and review the arguments to use during the build process.
+CanvOS requires arguments, such as image tag, registry, repository, and OS distribution, defined in the  **.arg** file. In this step, you will create the  **.arg** file and define all these arguments.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md:125]:
```suggestion
CanvOS requires arguments, such as image tag, registry, repository, and OS distribution. The arguments are defined in the  **.arg** file. In this step, you will create the  **.arg** file and define all the required arguments.
```
This reads smoother if you break it up into two sentences.

File: content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md [hunk 2]
```diff
@@ -120,17 +120,46 @@ git checkout v3.4.3
...
+
+Issue the command below to assign an image tag value for the provider images. This guide uses the default value `demo` as an example. However, you can assign any lowercase and alphanumeric string to the `CUSTOM_TAG` variable. 
+<br />
+
+```bash
+export CUSTOM_TAG=demo
+```
 <br />
 
+Issue the command below to create the **.arg** file with the custom tag. It uses the default values for all arguments. For example, `ubuntu` is the default operating system, `demo` is the default tag, and [ttl.sh](https://ttl.sh/) is the default image registry. This image registry is free to use and does not require a sign-up. Images pushed to *ttl.sh* are ephemeral and will expire after the 24 hrs time limit.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md:136]:
```suggestion
Issue the command below to create the **.arg** file with the custom tag. The remaining arguments will use the default values. For example, `ubuntu` is the default operating system, `demo` is the default tag, and [ttl.sh](https://ttl.sh/) is the default image registry. The default image registry, ttl.sh, is free to use and does not require a sign-up. Images pushed to ttl.sh are ephemeral and will expire after the 24 hrs time limit. 
```

File: content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md [hunk 3]
```diff
@@ -120,17 +120,46 @@ git checkout v3.4.3
...
+<br />
+
+```bash
+export CUSTOM_TAG=demo
+```
 <br />
 
+Issue the command below to create the **.arg** file with the custom tag. It uses the default values for all arguments. For example, `ubuntu` is the default operating system, `demo` is the default tag, and [ttl.sh](https://ttl.sh/) is the default image registry. This image registry is free to use and does not require a sign-up. Images pushed to *ttl.sh* are ephemeral and will expire after the 24 hrs time limit. 
+
+Using the arguments defined in the **.arg** file, the final provider images you generate will have the following naming convention, `[IMAGE_REGISTRY]/[IMAGE_REPO]:[CUSTOM_TAG]`. In this example, the provider images will be `ttl.sh/ubuntu:k3s-1.25.2-v3.4.3-demo`. To know the list of arguments you can define or find their default values, refer to the **.arg.template** sample file in the current directory.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md:138]:
```suggestion
Using the arguments defined in the **.arg** file, the final provider images you generate will have the following naming convention, `[IMAGE_REGISTRY]/[IMAGE_REPO]:[CUSTOM_TAG]`. In this example, the provider images will be `ttl.sh/ubuntu:k3s-1.25.2-v3.4.3-demo`. Refer to the **.arg.template** sample file in the current directory or the [README](https://github.com/spectrocloud/CanvOS#readme) to learn more about the default values. 
```

**Review by karl-cardenas-coding — APPROVED** [review]

File: content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md [hunk 4]
```diff
@@ -120,17 +120,46 @@ git checkout v3.4.3
 ```
 <br />
 
-## Review Arguments
+## Define Arguments
 
-The **.arg.template** sample file in the current directory contains customizable arguments, such as image tag, registry, repository, and OS distribution. Rename the sample **.arg.template** file to **.arg** and review the arguments to use during the build process.
+CanvOS requires arguments, such as image tag, registry, repository, and OS distribution. The arguments are defined in the  **.arg** file. In this step, you will create the  **.arg** file and define all the required arguments.
```
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md:125]:
```suggestion
CanvOS requires arguments such as image tag, registry, repository, and OS distribution. The arguments are defined in the  **.arg** file. In this step, you will create the  **.arg** file and define all the required arguments.
```

File: content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md [hunk 5]
```diff
@@ -120,17 +120,46 @@ git checkout v3.4.3
...
+
+Issue the command below to assign an image tag value for the provider images. This guide uses the default value `demo` as an example. However, you can assign any lowercase and alphanumeric string to the `CUSTOM_TAG` variable. 
+<br />
+
+```bash
+export CUSTOM_TAG=demo
+```
 <br />
 
+Issue the command below to create the **.arg** file with the custom tag. The remaining arguments will use the default values. For example, `ubuntu` is the default operating system, `demo` is the default tag, and [ttl.sh](https://ttl.sh/) is the default image registry. The default image registry, ttl.sh, is free to use and does not require a sign-up. Images pushed to ttl.sh are ephemeral and will expire after the 24 hrs time limit.
```
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md:136]:
```suggestion
Issue the command below to create the **.arg** file with the custom tag. The remaining arguments will use the default values. For example, `ubuntu` is the default operating system, `demo` is the default tag, and [ttl.sh](https://ttl.sh/) is the default image registry. The default ttl.sh image registry is free and does not require a sign-up. Images pushed to ttl.sh are ephemeral and will expire after the 24 hrs time limit. 
```

File: content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md [hunk 6]
```diff
@@ -571,7 +615,7 @@ Open a web browser and log in to [Palette](https://console.spectrocloud.com). Na
 ![A screenshot showing the VMs automatically registered with Palette. ](/tutorials/edge/clusters_edge_deploy-cluster_edge-hosts.png)
 
 
-If the three Edge hosts are not displayed in the Edge hosts tab, the automatic registration failed. In this happens, you can manually register hosts by clicking the **Add Edge Hosts** button and pasting the Edge host ID. Repeat this host registration process for each of the three VMs.
+If the three Edge hosts are not displayed in the Edge hosts tab, the automatic registration failed. If this happens, you can manually register hosts by clicking the **Add Edge Hosts** button and pasting the Edge host ID. Repeat this host registration process for each of the three VMs.
```
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md:618]:
```suggestion
If the three Edge hosts are not displayed in the **Edge hosts** tab, the automatic registration failed. If this happens, you can manually register hosts by clicking the **Add Edge Hosts** button and pasting the Edge host ID. Repeat this host registration process for each of the three VMs.
```

**Review by ritawatson — APPROVED** [review]
A couple of minor suggestions.

**Pull request merged**